registration-processor/core-processor/registration-processor-bio-dedupe-stage/src/main/java/io/mosip/registration/processor/biodedupe/stage/BioDedupeProcessor.java

```diff
@@ -435,6 +439,36 @@ private void postAbisIdentification(InternalRegistrationStatusDto registrationSt
 					registrationStatusDto.getRegistrationId(), BioDedupeConstants.ABIS_RESPONSE_NULL);
 
 		} else {
+			if (registrationType.equalsIgnoreCase(RegistrationType.UPDATE.name()) && matchedRegIds.size() == 1
+					&& biometricAutorejectEnable) {
+				Optional<String> optionalMatchedRegId = matchedRegIds.stream().findFirst();
+				String matchedRegId = optionalMatchedRegId.get();
+				String uin = idRepoService.getUinByRid(matchedRegId, utilities.getGetRegProcessorDemographicIdentity());
+				String updateUin = utilities.getUIn(registrationStatusDto.getRegistrationId(), registrationType,
+						ProviderStageName.BIO_AUTH);
+				if (StringUtils.equals(uin, updateUin)) {
+					registrationStatusDto
+							.setLatestTransactionStatusCode(RegistrationTransactionStatusCode.SUCCESS.toString());
+					object.setIsValid(Boolean.TRUE);
+					registrationStatusDto.setStatusCode(RegistrationStatusCode.PROCESSING.name());
+					registrationStatusDto.setStatusComment(StatusUtil.BIO_DEDUPE_SUCCESS.getMessage());
+					registrationStatusDto.setSubStatusCode(StatusUtil.BIO_DEDUPE_SUCCESS.getCode());
+					regProcLogger.info(LoggerFileConstant.SESSIONID.toString(),
+							LoggerFileConstant.REGISTRATIONID.toString(), registrationStatusDto.getRegistrationId(),
+							BioDedupeConstants.ABIS_RESPONSE_MATCHING);
+				} else {
+					registrationStatusDto
+							.setLatestTransactionStatusCode(RegistrationTransactionStatusCode.FAILED.toString());
+					object.setIsValid(Boolean.FALSE);
+					registrationStatusDto.setStatusCode(RegistrationStatusCode.REJECTED.name());
+					registrationStatusDto.setStatusComment(StatusUtil.UPDATE_PACKET_BIOMETRICS_NOT_FOUND.getMessage());

```

Comment:
This case is not about matching with someone else biometrics ? But message says biometrics not found ?

Reply:
resolved